test/file_test.rb

```diff
@@ -135,25 +135,28 @@
 
   describe '#retrieve!' do
     before do
-      @file_with_content = RemoteFiles::File.new('identifier', :content => 'content', :content_type => 'content_type')
+      @file_with_content = RemoteFiles::File.new('identifier', :content => 'content', :content_type => 'content_type', :populate_stored_in => true)
```

Comment:
Should there also be a test where `populate_stored_in` is `nil` since that seems like the default behavior?

Reply:
I updated the unit tests to use `populate_stored_in` when it is `nil` and another section for when it is `true`